### DISCUSSION TITLE
Question: How do I create a mock up of custom touch controls?

### DISCUSSION THREAD

**Discussion by evanpaul14** [redphx/better-xcloud#283] [Custom touch controls]
Maybe this is a stupid question but I'm not really sure how to create a mock up of my touch controls? I haven't found any directions on how to do this.

**Comment by redphx** [comment]
It's not stupid at all. I just need to find time to write the guide. You'll need to get familiar with [Chrome DevTools](https://developer.chrome.com/docs/devtools/) and JSON.

**Comment by redphx** [comment]
The guide will be here: https://github.com/redphx/better-xcloud/wiki/How-to-create-custom-touch-control-layout  

I'm still writing it. In the meantime you should check its requirements.

**Reply by evanpaul14** [reply]
Thank you, I’ll take a look.

**Reply by redphx** [reply]
The basic guide is done

**Comment by redphx** [comment]
Close & move to https://github.com/redphx/better-xcloud/discussions/285
